qiita_db/test/test_artifact.py

```diff
@@ -0,0 +1,374 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+from tempfile import mkstemp
+from datetime import datetime
+from os import close, remove
+from os.path import exists, join, basename
+from functools import partial
+
+import pandas as pd
+
+from qiita_core.util import qiita_test_checker
+from qiita_db.artifact import Artifact
+from qiita_db.exceptions import (QiitaDBArtifactCreationError,
+                                 QiitaDBArtifactDeletionError,
+                                 QiitaDBOperationNotPermittedError,
+                                 QiitaDBUnknownIDError)
+from qiita_db.metadata_template import PrepTemplate
+from qiita_db.study import Study
+from qiita_db.software import Command, Parameters
+from qiita_db.util import get_mountpoint, get_count
+
+
+@qiita_test_checker()
+class ArtifactTests(TestCase):
```

Comment:
From the point of view of the system is just a "processed" artifact, so it shouldn't make a difference. Anyways, I added a test. At the end is better to be safe than sorry.